package.json

```diff
@@ -1,57 +1,75 @@
-		"start": "react-scripts start",
-		"build": "react-scripts build",
-		"test": "react-scripts test --passWithNoTests",
-		"eject": "react-scripts eject"
-	},
-	"husky": {
-		"hooks": {
-			"pre-commit": "lint-staged"
-		}
-	},
-	"lint-staged": {
-		"*.{js,ts,tsx}": "eslint --cache --fix --ignore-path .eslintignore",
-		"*.{js,ts,tsx,css,md}": "prettier --write --ignore-path .eslintignore"
-	},
-	"eslintConfig": {
-		"extends": "react-app"
-	},
-	"browserslist": {
-		"production": [
-			">0.2%",
-			"not dead",
-			"not op_mini all"
-		],
-		"development": [
-			"last 1 chrome version",
-			"last 1 firefox version",
-			"last 1 safari version"
-		]
-	},
-	"devDependencies": {
-		"husky": "^4.2.5",
-		"lint-staged": "^10.2.6",
-		"prettier": "^2.0.5"
-	}
+    "name": "virtual-mmx",
+    "version": "0.1.0-alpha",
+    "private": true,
+    "description": "Web application simulating Wintergatans Marble Maschine X",
+    "main": "index.js",
+    "dependencies": {
+        "vmmx-schema": "wintergatan-community/vmmx-schema",
+        "@testing-library/jest-dom": "^4.2.4",
+        "@testing-library/react": "^9.5.0",
+        "@testing-library/user-event": "^7.2.1",
+        "@types/jest": "^24.9.1",
+        "@types/node": "^12.12.42",
+        "@types/react": "^16.9.35",
+        "@types/react-dom": "^16.9.8",
+        "bootstrap": "^4.5.0",
+        "react": "^16.13.1",
+        "react-bootstrap": "^1.0.1",
+        "react-dom": "^16.13.1",
+        "tone": "^13.8.25"
+    },
+    "devDependencies": {
+        "husky": "^4.2.5",
+        "lint-staged": "^10.2.6",
+        "prettier": "^2.0.5",
+        "@babel/core": "^7.9.6",
+        "@babel/plugin-proposal-class-properties": "^7.8.3",
+        "@babel/plugin-proposal-object-rest-spread": "^7.9.6",
+        "@babel/preset-env": "^7.9.6",
+        "@babel/preset-react": "^7.9.4",
+        "@babel/preset-typescript": "^7.9.0",
+        "@typescript-eslint/eslint-plugin": "^3.0.0",
+        "@typescript-eslint/parser": "^3.0.0",
+        "babel-loader": "^8.1.0",
+        "css-loader": "^3.5.3",
+        "eslint": "^7.1.0",
+        "eslint-config-prettier": "^6.11.0",
+        "eslint-plugin-prettier": "^3.1.3",
+        "eslint-plugin-react": "^7.20.0",
+        "html-webpack-plugin": "^4.3.0",
+        "style-loader": "^1.2.1",
+        "typescript": "^3.9.3",
+        "webpack": "^4.43.0",
+        "webpack-cli": "^3.3.11",
+        "webpack-dev-server": "^3.11.0"
+    },
+    "scripts": {
+        "bundle": "webpack",
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "start": "webpack-dev-server --mode development --open --hot",
+        "build": "webpack --mode production",
+        "check-types": "tsc"
+    },
+    "husky": {
+        "hooks": {
+            "pre-commit": "lint-staged"
+        }
+    },
+    "lint-staged": {
+        "*.{js,ts,tsx}": "eslint --cache --fix --ignore-path .eslintignore",
+        "*.{css,md}": "prettier --write --ignore-path .eslintignore"
```

Comment:
prettier should apply to js/jsx/tsx too- 
```
	"lint-staged": {
		"*.{js,ts,tsx}": "eslint --cache --fix --ignore-path .eslintignore",
		"*.{js,ts,tsx,css,md}": "prettier --write --ignore-path .eslintignore"
	},
```

Reply:
This plugin `"plugin:prettier/recommended"` in `.eslintrc` starts prettier with eslint.
So prettier indirectly applies to js/jsx/tsx.

Reply:
Okay, wasn't aware of that feat. Cool!